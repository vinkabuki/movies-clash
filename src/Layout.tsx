import React from 'react'

import styled from 'styled-components'

import Header from './Header'
import Body from './Body'
import Footer from './Footer'

const LayoutBox = styled.div`
  max-width: 400px;
  margin: auto;
  text-align: center;
`

const Layout: React.FC = () => {
  return (
    <LayoutBox>
      <Header />
      <Body />
      <Footer />
    </LayoutBox>
  )
}

export default Layout
