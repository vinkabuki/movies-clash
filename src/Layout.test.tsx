import React from 'react'
import renderer from 'react-test-renderer'
import Layout from './Layout'

describe('Layout component basic test', () => {
  test('Render Layout component', () => {
    const component = renderer.create(<Layout />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
