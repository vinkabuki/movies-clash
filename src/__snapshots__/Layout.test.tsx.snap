// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component basic test Render Layout component 1`] = `
<div
  className="sc-fzplWN bupGwC"
>
  <div>
    <h2>
      Movies Clash
      <span>
        <img
          className="sc-AxjAm dYgbqu"
          src="logo.jpg"
        />
      </span>
    </h2>
  </div>
  <div>
    rendering
  </div>
  <p
    className="sc-fzpans bYcObV"
  >
    Thank you for using Movies Clash
  </p>
</div>
`;
